gazebo_plugins/src/gazebo_ros_force.cpp

```diff
@@ -43,6 +43,9 @@ class GazeboRosForcePrivate
 
   // Pointer to the update event connection
   gazebo::event::ConnectionPtr update_connection_;
+
+  /// Indicates that the force should be applied on the world frame instead of the body frame
+  bool force_world_frame_;
```

Comment:
This variable is only used within the `Load` function, so we don't need to store it as a member variable.